packages/@magic-sdk/types/src/modules/user-types.ts

```diff
@@ -16,6 +16,11 @@ export interface GenerateIdTokenConfiguration extends GetIdTokenConfiguration {
   attachment?: string;
 }
 
+export enum AllowAllEvents {
+  ClosedByUser = 'closed-by-user',
+  ClosedByUserHandle = 'closed-by-user-handle',
```

Comment:
What's the difference between `ClosedByUser` and `ClosedByUserHandle`

Reply:
Also curious on this